src/ActiveLogin.Authentication.BankId.Api/Models/SignRequest.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace ActiveLogin.Authentication.BankId.Api.Models
+{
+    /// <summary>
+    /// Sign request parameters.
+    /// </summary>
+    [DataContract]
+    public class SignRequest : IAuthRequest
```

Comment:
I see an issue with adding/removing parameters in the middile when they are all of the same type (personalIdentity Number).

Suggesting these overloads:

- `public SignRequest(string endUserIp, string userVisibleData)`
- `public SignRequest(string endUserIp, string userVisibleData, string userNonVisibleData)`
- `public SignRequest(string endUserIp, string userVisibleData, string userNonVisibleData, string personalIdentityNumber)`
- `public SignRequest(string endUserIp, string userVisibleData, string userNonVisibleData, string personalIdentityNumber, Requirement requirement)`